pkgs/tools/misc/iay/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, nixosTests
+, openssl
+, pkg-config
+, Security
+, Foundation
+, Cocoa
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "iay";
+  version = "0.4.0";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    Security
+    Foundation
+    Cocoa
+  ]
+  ++ [ openssl pkg-config ]
```

Comment:
1. pkg-config goes on `nativeBuildInputs`
2. the list of optionals should be last on this concatenation.

Reply:
Sure.